### PR TITLE
Update to new issue template system.

### LINKED ISSUE #1211: Overhaul Issue Template
As discussed at the meeting please complete following tasks
- @jaj22 provide a list of bulletpoints of what information is required
- @SonorousMeerkat once John has supplied the list, change it and write it in an appropriate way.

### PR DESCRIPTION
**Pull request by SonorousMeerkat**
Adds Bug Report, Feature request and change request template. The latter two are functionally the same, but allow us to separate them for organisation sake.

closes #1211

### CODE REVIEW

**Comment by ComboTombo** [comment]
Looks pretty neat, not gonna lie.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+4. 
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+Version:
```
Comment (jaj22) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Need to make it clear that this should include the Antistasi version number, map and greenfor/blufor.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+Version: 
+    
+
+[ ] LAN Hosted
+[ ] Private dedicated server
```
Comment (jaj22) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Should add dedicated server with HC(s) under this.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the bug:
+1. 
+2. 
+3. 
+4.
```
Comment (jaj22) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: While there is an argument for dissuading people from reporting bugs without reproduction steps, some clear bugs will be very difficult for users to reproduce. We could add an alternative option of "If you don't know how to reproduce this bug, try to describe the relevant history leading up to the bug".

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+[ ] LAN Hosted
+[ ] Private dedicated server
+[ ] Community Server 1
+[ ] Community Server 2
+[ ] Community Test Server
+[ ] Community Test Server 2
+
+Modset:
+    
+
```
Comment (jaj22) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: If the bug was experienced on the community servers and isn't easily reproducible, we'd also need the (real world) time of the incident and potentially the player's in-game name. Similarly if it happened on a private server and they provided the RPT. Could move the "please provide RPT" text down here as well.

**Pull request merged**